### PR TITLE
Create victini.ms

### PR DESCRIPTION
**Pull request by ThisChicken**
Please, you added Mew and Mewtwo. Why not?

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/victini.ms [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                "/flyspeed <number>",
+                "/fly",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/tp",
+                "/speak <playername>",
+                "/jump 0-7",
+                "/run 0-5",
+                "/confuse <player>",
+                "/haste 0-4"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/victini.ms]: missing a comma here

File: CommandHelper/LocalPackages/commands/buffs/victini.ms [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                "/vis           
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 4 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/victini.ms]: What makes Victini stronger than the strongest human?

File: CommandHelper/LocalPackages/commands/buffs/victini.ms [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                "/effect ". @plr ." 11 9999 9 true",
+                "/effect ". @plr ." 21 9999 4 true"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/victini.ms]: What makes Victini so durable?

File: CommandHelper/LocalPackages/commands/buffs/victini.ms [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                "/fly",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/tp",
+                "/speak <playername>",
+                "/jump 0-7",
+                "/run 0-5",
+                "/confuse <player>",
+                "/haste 0-4",
+                "/vis
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/victini.ms]: missing " here

**Comment by jaikev** [comment]
And why /vis he can't do it in the movie

**Pull request closed without merging**